modules/allometry/R/allom.BayesFit.R

```diff
@@ -265,9 +263,9 @@ allom.BayesFit <- function(allom, nrep = 10000, form = "power", dmin = 0.1, dmax
         u2 <- u2 + 0.5 * crossprod(Y - X %*% beta)
```

Comment:
```suggestion
        u2 <- u2 + as.numeric(0.5 * crossprod(Y - X %*% beta))
```

Reply:
@Aariq I think this will fix the warning you identified. Is this the only one or were there other similar warnings?